### ISSUE TITLE
Standardize Celestia icon

### ISSUE THREAD

**Issue by SevenSpheres** [CelestiaProject/Celestia#507]
[This icon](https://github.com/CelestiaProject/Celestia/blob/master/celestia-logo.png) should be used in all front ends.

**Comment by eyvallah** [comment]
yes, 1.6.x branch on Mac has changed to use this icon now, we should replace the current icon on other platforms too. generating an icon file should be trivial

**Comment by Panterstruck** [comment]
Sigh, still sad to see the icon I was forced to make used but let me know if you need my icon file or higher res files. Or ideally use a better quality redesign of that icon.

But yeah I agree, would be nice if the actual appication used the icon too.

**Comment by Anthony-B-Russo10** [comment]
FarGetaNik, why don't you try and use your icon. https://celestia.space/forum/viewtopic.php?f=5&t=19914&p=146117&hilit=icon#p146117

**Comment by SevenSpheres** [comment]
I actually prefer the icon that is obviously Saturn to the "simple" one.

**Comment by Anthony-B-Russo10** [comment]
With the way IS GUIs are now, that simple icon is a better fit then the current Windows XP and OS X 10.5 Leopard icons.

**Comment by LafuenteAstronomy** [comment]
How about we just make the circle in the center the Sun? Then the "C" will just be greyish It'll be much simpler than the Saturn one, which I understand why Pantherstruck does not like seeing

**Comment by LafuenteAstronomy** [comment]
Which just so happens to look like the icon in the link, but just make it more color-accurate

**Comment by SevenSpheres** [comment]
Well if more people like the "simplified" icon, that can be used instead. The point is to use the same icon in all frontends. Maybe we can have a poll for which icon to use.

**Comment by Panterstruck** [comment]
I did refine that icon a bit more for Project Echoes 0.2:

![celestia-c13](https://user-images.githubusercontent.com/57594937/68997612-b4dafd80-08a8-11ea-9124-e7a98b8a3c12.png)

Hm i can't upload icon files directly.

A star in the center might work too, but iirc thats very similar to Celestia Origins icon.

**Comment by SevenSpheres** [comment]
How do I make a poll on the forum?

**Comment by Panterstruck** [comment]
Good question. Maybe only admins can?

**Comment by SevenSpheres** [comment]
You know what, I do like the new icon. I still want to make a poll though.

**Comment by SevenSpheres** [comment]
You know what, I do like the new icon. I still want to make a poll though.

**Comment by Panterstruck** [comment]
Glad to hear that. It's just my current result tho. Importantly I think the icon should not feature texture details or harsh shading like th current one, anything else im up for debate. Just one thing that many have commented on, "realistic" colored Saturn looks too bland as an icon. You'd want some contrast in colors.

**Comment by SevenSpheres** [comment]
"Wishlist"? IMO this is something that needs to be done for the 1.7.0 release.
